src/Database/ODBC/Internal.hs

```diff
@@ -360,7 +360,7 @@ streamWithParams ::
   -> (state -> [(Column, Value)] -> m (Step state))
```

Comment:
```suggestion
  -> (state -> [Value] -> m (Step state))
```

Given that `state` is now derived from our columns, does it make sense to remove `Column` here?